dataverse-webapp/src/main/webapp/dataset-license-terms.xhtml

```diff
@@ -5,319 +5,55 @@
-                            </label>
-                            <div class="col-sm-9">
-                                <p:selectBooleanCheckbox id="requestAccess" itemLabel="#{bundle['file.dataFilesTab.terms.list.termsOfAccess.requestAccess.enableBtn']}" value="#{termsOfUseAndAccess.fileAccessRequest}"/>
-                            </div>
-                        </div>
-                    </ui:fragment>
-
-                    <!-- Additional Information
-                    <ui:fragment rendered="# {!empty editMode or empty editMode and (!empty termsOfUseAndAccess.dataAccessPlace
-                                         or !empty termsOfUseAndAccess.originalArchive or !empty termsOfUseAndAccess.availabilityStatus
-                                         or !empty termsOfUseAndAccess.contactForAccess or !empty termsOfUseAndAccess.sizeOfCollection
-                                         or !empty termsOfUseAndAccess.studyCompletion)}"> -->
+    <!-- Guestbook -->
+        <div class="panel panel-default" jsf:rendered="#{!settingsWrapper.rsyncOnly}">
+            <div class="panel-body">
+                <label for="metadata_Guestbook" class="col-sm-3 control-label">
+                    #{bundle['file.dataFilesTab.terms.list.guestbook']}
+                    <span class="glyphicon glyphicon-question-sign tooltip-icon"
+                          data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['file.dataFilesTab.terms.list.guestbook.title']}"></span>
+                </label>
+                <div class="col-sm-9">
+                    <ui:fragment>
+                        <ui:fragment rendered="#{empty GuestbookTab.selectedGuestbook
+                                                and ((!dataverseSession.user.authenticated or !GuestbookTab.canIssueUpdateDatasetCommand)
+                                                or ((dataverseSession.user.authenticated and GuestbookTab.canIssueUpdateDatasetCommand
+                                                   and GuestbookTab.dataverseGuestbooksCount gt 0)))}">
+                            <p>#{bundle['file.dataFilesTab.terms.list.guestbook.noSelected.tip']}</p>
+                        </ui:fragment>
+                        <ui:fragment rendered="#{empty GuestbookTab.selectedGuestbook
+                                         and (dataverseSession.user.authenticated and GuestbookTab.canIssueUpdateDatasetCommand)
+                                         and GuestbookTab.dataverseGuestbooksCount == 0}">
+                            <p>
+                                <h:outputFormat value="#{bundle['file.dataFilesTab.terms.list.guestbook.noSelected.admin.tip']}">
+                                    <f:param value="#{GuestbookTab.ownerName}"/>
+                                </h:outputFormat>
+                            </p>
+                        </ui:fragment>
```

Comment:
I would refactor conditions on this fragment and the one above to something like:
```
<ui:fragment rendered="#{!GuestbookTab.canIssueUpdateDatasetCommand}">
  file.dataFilesTab.terms.list.guestbook.noSelected.tip message
</ui:fragment>
<ui:fragment rendered="#{GuestbookTab.canIssueUpdateDatasetCommand}">
  <p jsf:rendered="#{GuestbookTab.dataverseGuestbooksCount gt 0}">
    file.dataFilesTab.terms.list.guestbook.noSelected.tip message
  </p>
 <p jsf:rendered="#{GuestbookTab.dataverseGuestbooksCount == 0}">
   file.dataFilesTab.terms.list.guestbook.noSelected.admin.tip message
 </p>
</ui:fragment>
```
For this to work we must also address to previous cr comment.
We skipped checking if user is authenticated because it shouldn't concern us. Checking `canIssueUpdateDatasetCommand` is enough